### ISSUE TITLE
'Remove' option (and others) not available on context menu sometimes

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#6852]
### Description

![image](https://user-images.githubusercontent.com/1190097/41126305-4ae6e1e8-6a64-11e8-998f-1c4dbf48f24e.png)

### Steps to reproduce

- Send a message in a room
- Try to redact it
- Get some coffee and re-open the context menu
- See the remove option has magically appeared

I'm not sure if it's time based, state based, or something else.

Can only reproduce on Desktop for my own messages. Other people's work as expected.

### Version information (not my usual client)

- **Platform**: desktop - cannot reproduce on Chrome 66 web
- **OS**: Windows 10
- **Version**: 0.15.4

**Comment by t3chguy** [comment]
I have seen this to and could not figure out what caused it at the time :(

**Comment by turt2live** [comment]
It's 100% reproducible when I send a message (waiting for it to go to 'sent'). However by the time I get a chance to debug it the problem goes away :/

**Comment by t3chguy** [comment]
yup so this is because a null event status is after remote-echo, but we have the event-id before then so we can do things when =="SENT"
